RRemoteConfig/Bundle+Environment.swift

```diff
@@ -0,0 +1,27 @@
+extension Bundle: EnvironmentSetupProtocol {
+    var valueNotFound: String {
+        return "NONE"
+    }
+
+    func deviceOsVersion() -> String {
+        return UIDevice.current.systemVersion
+    }
+
+    func deviceModel() -> String {
+        var systemInfo = utsname()
+        uname(&systemInfo)
+        return String(bytes: Data(bytes: &systemInfo.machine, count: Int(_SYS_NAMELEN)), encoding: .ascii)!.trimmingCharacters(in: .controlCharacters)
+    }
+
+    func sdkName() -> String {
+        return "Remote Config"
+    }
+
+    func sdkVersion() -> String {
+        return Bundle(for: Environment.self).value(for: "CFBundleShortVersionString") ?? self.valueNotFound
```

Comment:
I don't really understand how this works. Why does `CFBundleShortVersionString` here return the SDK version instead of the App version? Is it because of the `for: Environment.self`?

Reply:
Yes. Environment is a class in the RemoteConfig.framework so getting the bundle for that class will return the framework's bundle.